packages/components/src/highlight-cards/style.scss

```
@@ -24,10 +24,13 @@ $vertical-margin: 32px;
 .highlight-cards-list {
 	display: flex;
 	flex-flow: row nowrap;
+	max-width: 100%;
+	overflow-x: scroll;
+	padding: 1px; // Ensures scrollable content box doesn't cover the highlight card.
 
 	.highlight-card {
 		border-color: var(--studio-gray-5);
-		border-radius: 4px;
+		border-radius: 5px; // stylelint-disable-line scales/radii
 		width: 100%;
 		min-width: 200px; // Minimum mobile width
```

```suggestion
		min-width: 200px; // Minimum mobile width
		padding: 16px 24px;
		margin-bottom: 0;
```

Some styles are not precisely the same as the design, which would not be a blocker. 🚀